src/test/enhetstester/kotlin/no/nav/familie/ba/sak/kjerne/fagsak/FagsakDeltagerServiceTest.kt

```diff
@@ -189,4 +197,74 @@ class FagsakDeltagerServiceTest {
         assertThat(resultat.single { it.ident == morIdent }.erEgenAnsatt).isTrue()
         assertThat(resultat.single { it.ident == farIdent }.erEgenAnsatt).isNull()
     }
+
+    @Test
+    fun `skal omgjøre alle exceptions unntat Funksjonell feil til Feil`() {
```

Comment:
Sykeste småpirk men unntatt
```suggestion
    fun `skal omgjøre alle exceptions unntatt Funksjonell feil til Feil`() {
```